### DISCUSSION TITLE
Beatmap playback menu can be open while editing a skin, covering skinning elements

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#31015] [Ideas] [answered]
When a beatmap is being autoplayed and you have the skin editor open, you can still pull out the autoplay menu from the right side of the screen. While the menu itself is inoperable (any clicks are sent through to the elements behind the menu buttons), it still causes visual clutter and makes it hard to see some elements, especially near the upper right corner where the play/pause controls are. This issue was observed when accessing the skin editor from the main menu, as well as when turning on autoplay mod and choosing a song from the song select.

Since the menu itself cannot be used while editing, it would be nice if it could not appear at all when the editor is open.

![image](https://github.com/user-attachments/assets/24a86858-66a1-4e93-a417-422f325d485a)

This menu of course does not appear at all if you are not using autoplay, but obviously skin editing is a lot harder without it since you can still fail songs.

**Comment by peppy** [comment, marked as answer]
Fixed in the next release.

**Reply by Joehuu** [reply]
https://github.com/ppy/osu/pull/31105 for reference.
